src/app/process-editor/config/palette-elements.json

```diff
@@ -0,0 +1,109 @@
+[
+    {
+        "group": "tool",
+        "type": "hand-tool",
+        "icon": "bpmn-icon-hand-tool",
+        "title": "APP.PROCESS_EDITOR.PALETTE.HAND_TOOL",
+        "clickable": true,
+        "draggable": false
+    },
+    {
+        "group": "tool",
+        "type": "connect-tool",
+        "icon": "bpmn-icon-connection-multi",
+        "title": "APP.PROCESS_EDITOR.PALETTE.CONNECT_TOOL",
+        "clickable": true,
+        "draggable": false
+    },
+    {
+        "group": "tool",
+        "type": "space-tool",
+        "icon": "bpmn-icon-space-tool",
+        "title": "APP.PROCESS_EDITOR.PALETTE.SPACE_TOOL",
+        "clickable": true,
+        "draggable": false
+    },
+    {
+        "group": "tool",
+        "type": "lasso-tool",
+        "icon": "bpmn-icon-lasso-tool",
+        "title": "APP.PROCESS_EDITOR.PALETTE.LASSO_TOOL",
+        "clickable": true,
+        "draggable": false
+    },
+    { "group": "separator" },
+    {
+        "group": "element",
+        "type": "bpmn:StartEvent",
+        "icon": "bpmn-icon-start-event-none",
+        "title": "APP.PROCESS_EDITOR.PALETTE.START_EVENT",
+        "clickable": true,
+        "draggable": true
+    },
+    {
+        "group": "element",
+        "type": "bpmn:EndEvent",
+        "icon": "bpmn-icon-end-event-none",
+        "title": "APP.PROCESS_EDITOR.PALETTE.END_EVENT",
+        "clickable": true,
+        "draggable": true
+    },
+    { "group": "separator" },
+    {
+        "group": "element",
+        "type": "bpmn:Gateway",
+        "icon": "bpmn-icon-gateway-none",
+        "title": "APP.PROCESS_EDITOR.PALETTE.GATEWAY",
+        "clickable": true,
+        "draggable": true
+    },
+    { "group": "separator" },
+    {
+        "group": "element",
+        "type": "bpmn:UserTask",
+        "icon": "bpmn-icon-user-task",
+        "title": "APP.PROCESS_EDITOR.PALETTE.USER_TASK",
+        "clickable": true,
+        "draggable": true
+    },
+    {
+        "group": "element",
+        "type": "bpmn:ServiceTask",
+        "icon": "bpmn-icon-service-task",
+        "title": "APP.PROCESS_EDITOR.PALETTE.SERVICE_TASK",
+        "clickable": true,
+        "draggable": true
+    },
+    {
+        "group": "element",
+        "type": "bpmn:CallActivity",
+        "icon": "bpmn-icon-call-activity",
+        "title": "APP.PROCESS_EDITOR.PALETTE.USER_TASK",
+        "clickable": true,
+        "draggable": true
+    },
+    {"group": "separator" },
```

Comment:
From this line down, I assume we don't need those elements at the final product, they are just dummies.